tools/ExtensionLibrary/DevHome.ExtensionLibrary/ViewModels/ExtensionLibraryViewModel.cs

```diff
@@ -10,4 +14,10 @@ public partial class ExtensionLibraryViewModel : ObservableObject
     public ExtensionLibraryViewModel()
     {
     }
+
+    [RelayCommand]
+    public async Task GetUpdatesButton()

```

Comment:
```suggestion
    public async Task GetUpdatesButtonAsync()
```

No need to change the corresponding binding as MVVM toolkit will auto remove the Async suffix 🙂